dot/rpc/modules/state_integration_test.go

```diff
@@ -28,29 +28,35 @@ const (
 )
 
 func TestStateModule_GetRuntimeVersion(t *testing.T) {
-	// expected results based on responses from prior tests
+	/* expected results based on responses from prior tests
+	We can get this data from polkadot runtime release source code
+	https://github.com/paritytech/polkadot/blob/v0.9.29/runtime/westend/src/lib.rs#L105-L117
+	*/
 	expected := StateRuntimeVersionResponse{
-		SpecName:         "node",
-		ImplName:         "substrate-node",
-		AuthoringVersion: 10,
-		SpecVersion:      264,
+		SpecName:         "westend",
```

Comment:
I think this changes are in develop now, you can merge from there

Reply:
Unresolving this, since it's still shows as changes, this branch needs a rebase on development

Reply:
I've replace this PR with PR #3192 since this PR contains code from other branches.

Reply:
So can we close this PR and continue in #3192 ?